developer-guides/server-side.mdx

```diff
@@ -83,13 +83,15 @@ app.get("/", (req, res) => {
 ### Sending events
 
 You can also send custom events to Koala using the same `/batch` endpoint for your workspace.
+The API call must include either an `email` (most common scenario), or a `profile_id` (read from the Koala cookie).
```

Comment:
Note: this is already specified above!

Reply:
Yes. I added it a second time because the FAL person didn’t see it the first time.

also made the top one bold lol